android/src/main/java/com/tailscale/ipn/ui/view/MainView.kt

```diff
@@ -597,6 +615,38 @@ fun PeerList(
                                     shape = RoundedCornerShape(percent = 50))) {}
                     Spacer(modifier = Modifier.size(8.dp))
                     Text(text = peer.displayName, style = MaterialTheme.typography.titleMedium)
+                    DropdownMenu(
```

Comment:
Long press doesn't work on Android TV.   We should provide a way to access ping from the details view or just embed the composable right in the details view and show your ping times automatically.

Reply:
Added to the peer details view as well, with a button in the top bar.